sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/CommunicationTokenRefreshOptions.java

```diff
@@ -42,6 +46,25 @@ public CommunicationTokenRefreshOptions(Supplier<Mono<String>> tokenRefresher, b
         this.tokenRefresher = tokenRefresher;
         this.refreshProactively = refreshProactively;
         this.initialToken = initialToken;
+        this.refreshTimeBeforeTokenExpiry = getDefaultRefreshTimeBeforeTokenExpiry();
```

Comment:
Nit: you can call another constructor using
`this(tokenRefresher, refreshProactively, initialToken, getDefaultRefreshTimeBeforeTokenExpiry());`